src/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -92,7 +104,16 @@ export default function VideoPlayer(
       {errored ? (
         <span dangerouslySetInnerHTML={{ __html: errorMessage }} />
       ) : (
-        embedCode
+        <>
+          {headingText && <Heading level={2} text={headingText} />}
+          <p>{descriptionText}</p>
```

Comment:
This should also be a conditional since `descriptionText` is optional. Otherwise, a `<p>` tag will always be rendered even if no `descriptionText` was passed.

Reply:
Good catch.  A foolish oversight on my part.  Conditional logic has been added.